wrappers/c/zxing.h

```diff
@@ -0,0 +1,143 @@
+#ifndef _ZXING_C_H
+#define _ZXING_C_H
+
+/*
+ * <stdbool.h>
+ *  bool
+ *
+ * <stdint.h>
+ *  uint8_t
+ */
+#include <stdbool.h>
+#include <stdint.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/*
+ * ZXing/ImageView.h
+ */
+
+typedef void * zxing_ImageView;
+
+typedef enum {
+	zxing_ImageFormat_None = 0,
+	zxing_ImageFormat_Lum  = 0x01000000,
+	zxing_ImageFormat_RGB  = 0x03000102,
+	zxing_ImageFormat_BGR  = 0x03020100,
+	zxing_ImageFormat_RGBX = 0x04000102,
+	zxing_ImageFormat_XRGB = 0x04010203,
+	zxing_ImageFormat_BGRX = 0x04020100,
+	zxing_ImageFormat_XBGR = 0x04030201,
+} zxing_ImageFormat;
+
+zxing_ImageView zxing_ImageView_new (const uint8_t * data, int width, int height, zxing_ImageFormat format, int rowStride, int pixStride);
+void zxing_ImageView_free (zxing_ImageView iv);
+
```

Comment:
`new` and `free` is a wired mix of c++ and c terminology. I would suggest to use ether `new` and `delete` or `create` and `destroy`.